app/models/query/results/group_by.rb

```diff
@@ -163,7 +163,7 @@ def aliased_group_by_sort_order(alias_name, sortable, columns_hash, order = nil)
     column = case_insensitive_condition(sortable, column, columns_hash)
 
     if order
-      column + " #{order} "
+      column.to_s + " #{order} "
```

Comment:
Will this work:
```suggestion
      "#{column} #{order} "
```
Also wondering if space is needed at the end:
```suggestion
      "#{column} #{order}"
```